mne/connectivity/spectral.py

```diff
@@ -413,10 +415,6 @@ def _epoch_spectral_connectivity(data, sig_idx, tmin_idx, tmax_idx, sfreq,
     else:
         psd = None
 
-    # tell the methods that a new epoch starts
-    for method in con_methods:
-        method.start_epoch()
-
```

Comment:
I will put it back for safety reasons as it would change the API contract, even if our estimarors don't implement it.
Anyway I realize that for amplitude estimators we would need some pattern in which compute con is called directly in each epoch instead of accumulate but nothing happens in the end, no compute_con, only averaging.